sdk/videoanalyzer/video-analyzer-edge/samples/javascript/README.md

```diff
@@ -0,0 +1,31 @@
+# Azure Video Analyzer Edge client library for JavaScript
```

Comment:
Are you writing these by hand or autogenerated by `dev-tool samples publish`? If you haven't already, please follow the Samples v2 migration guide to make sure your samples follow the new model, which will help having them in sync

https://github.com/Azure/azure-sdk-for-js/wiki/Samples-v2-Migration-Guide

Reply:
Looking at the Samples v2 migration guide it requires from public docs that we have not yet published so can we do the migration for the GA release?